### PR TITLE
Changes to package.json and renderer.js

### PR DESCRIPTION
**Pull request by richardfazzio**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -18,10 +18,9 @@
   "author": "Roger Urscheler <roger.urscheler@unify.com>",
   "license": "Apache-2.0",
   "dependencies": {
-    "circuit-sdk": "^1.2.1900",
-    "electron": "^1.8.1"
+    "circuit-sdk": "^1.2.3302"
```
Comment (rogeru) [inline package.json]: Change to the latest sdk which is currently 1.2.3503

File: package.json [hunk 2]
```diff
@@ -18,10 +18,9 @@
...
   },
   "devDependencies": {
-    "electron": "^1.7.6"
+    "electron": "^2.0.2"
```
Comment (rogeru) [inline package.json]: Change to the latest version which is 2.0.4

**Review by rogeru — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (richardfazzio) [reply package.json]: done

File: package.json (hunk 2 above)
Comment (richardfazzio) [reply package.json]: done

File: package.json [hunk 3]
```diff
@@ -18,10 +18,9 @@
   "author": "Roger Urscheler <roger.urscheler@unify.com>",
   "license": "Apache-2.0",
   "dependencies": {
-    "circuit-sdk": "^1.2.1900",
-    "electron": "^1.8.1"
+    "circuit-sdk": "^1.2.3503"
   },
   "devDependencies": {
-    "electron": "^1.7.6"
+    "electron": "^2.0.4"
```
Comment (rogeru) [inline package.json]: This is wrong. electron needs to be under "dependencies".

**Review by rogeru — CHANGES_REQUESTED** [review]

**Pull request merged**